tools/build_browser.js

```diff
@@ -254,14 +248,71 @@ async function buildBrowserHighlightJS(languages, { minify }) {
   return {
     core: coreSize,
     core_min: core_min,
-    minified: Buffer.byteLength(minifiedSrc, 'utf8'),
+    minified: minifiedSrc.length,
```

Comment:
Is there a reason this was changed?

Reply:
`minifiedSrc` is now a `UInt8Array` in this implementation, not a string. So they should be exactly the same, expect that it should take a bit less time to compute. I could use `minifiedSrc.byteLength` to make it clearer – or I could simply revert this bit if you prefer the previous way of doing this.

Reply:
Yeah we don't need the encode... but we may not need bytelength anymore either, see my previous comment...